src/song.cpp

```diff
@@ -160,6 +167,13 @@ void		Song::init()
 		year = "";
 	}
 
+	/* original year from original date */
```

Comment:
I've often wished that the original year would display instead of the re-release year, so that doesn't apply to me unfortunately ;)
To resolve this, it could be implemented as an option. I'm not sure how much effort should be put into this old codebase though, with the new Go version is under development. I'll accept it as it is.